src/commands/bans/whobanned.ts

```diff
@@ -0,0 +1,17 @@
+
+import {Command} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+import {getBanRequester as whoBannedImpl, getBannedWord as bannedWordImpl} from "../../controllers/BanController";
+
+const whoBanned: Command = {
+    name: "whobanned",
+    "description": "Shows the username of the person who has banned a word",
+    usage: "whobanned",
+    procedure: async (client: Client, message: Message) => {
+        const bannedWord = await bannedWordImpl();
+        const reply = bannedWord != "" ? `${await whoBannedImpl()} has requested the ban` : `No word banned`;
```

Comment:
I could include the banned word in the reply, but I would need to make use of `banned.ts#censorWord`.

I think `censorWord` would logically belong better under `util`, but I didn't want to make that change in this PR. Let me know what you think.

Reply:
Will probably have one extra case: if the ban requester can't be found in the guild. I don't know if the client can arbitrarily mention anyone on all of discord with a uuid? Also the db _could_ return undefined.

Reply:
You'll need to test this but you might be able to just do
```typescript
`<@!${id}>`
```